org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/java/org/eclipse/winery/bpmn2bpel/BPMN4Tosca2BpelTest.java

```diff
@@ -13,50 +13,178 @@
  *******************************************************************************/
 package org.eclipse.winery.bpmn2bpel;
 
-import java.net.MalformedURLException;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.nio.file.Paths;
-
 import org.eclipse.winery.bpmn2bpel.parser.ParseException;
 import org.eclipse.winery.bpmn2bpel.planwriter.PlanWriterException;
-
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipException;
+import java.util.zip.ZipFile;
+
+import static org.junit.Assert.fail;
 
 public class BPMN4Tosca2BpelTest {
 
 	protected static String RESOURCES_DIR = "src/test/resources/bpmn4tosca";
 
+	/**
+	 * Temporary Folder instance to keep out Zips from Sources.
+	 */
+	public TemporaryFolder folder;
 
 	@Before
 	public void setUp() throws Exception {
-
+		folder = new TemporaryFolder(new File (RESOURCES_DIR));
+		folder.create();
+		folder.newFolder();
 	}
 
 	@After
 	public void tearDown() throws Exception {
+		folder.delete();
```

Comment:
According to the documetnation (http://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html), this is not necassary.